.ci_support/linux_64_.yaml

```diff
@@ -54,7 +54,7 @@ libcurl:
 libdeflate:
 - '1.24'
 libgdal:
-- '3.10'
+- '3.11'
```

Comment:
That looks wrong... given this PR is for the v3.10.x branch

Reply:
It is just how conda-forge pinning works.  My understanding from this happening before is that it is harmless...

Reply:
ok, thanks for the initiation :-)